### DISCUSSION TITLE
Are ESA income parameters named/described correctly?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-uk#61] [Q&A] [answered]
e.g. the description for https://github.com/PSLmodels/openfisca-uk/blob/master/openfisca_uk/parameters/benefits/ESA/income/amount_18_24.yaml is

https://github.com/PSLmodels/openfisca-uk/blob/eb284a7fbbbc92c8ae3d651b34667b39774e59a6/openfisca_uk/parameters/benefits/ESA/income/amount_18_24.yaml#L1

Is it the benefit amount or personal allowance?

**Comment by nikhilwoodruff** [comment, marked as answer]
This is the 'basic amount', but I agree it's a bit confusing. Some benefit sites refer to the basic amount as the 'personal allowance', e.g. [here](https://www.turn2us.org.uk/Benefit-guides/Employment-and-Support-Allowance/How-much-Employment-and-Support-Allowance-will-I-g). Most benefits follow the structure personal_allowance + premiums - deductions, and they're mainly consistent I think in referring to the basic amount as the personal allowance even when it might have more common names, so I guess it's a question of whether we want to be most similar to that specific benefit, or maintain more consistency across all benefits. But either way I think you're right, the filename should ideally be the same, rather than having this 'amount' vs 'allowance' ambiguity.
